subcommittees/mentoring/microLessonProposal.md

```diff
@@ -0,0 +1,36 @@
+## Micro-Lessons: Building Positive First Experiences in New Software Carpentry Instructors
+
+### Overview
+
+It has been observed [reference?] that a large fraction [number?] of people completing Software Carpentry (SWC) Instructor Training haven't yet taught a workshop, possibly due in part to the logistical challenges of first teaching with an experienced instructor. Furthermore, a great deal of the anonymous minute-card feedback collected after live Instructor Training events has reported misgivings felt by students on their level of familiarity with SWC material, and a lack of confidence that they were ready to teach an entire unit at a full workshop. Given the high demand for SWC workshops, the Mentorship Committee suspects the disconnect between Instructor Training and getting in front of a classroom may also be one of instructor confidence and experience, and may bridged by a smaller, guided teaching experience that serves as a stepping stone to a full workshop, and helps relax the logistical challenges of co-teaching with an experienced instructor.
```

Comment:
On Mon, Mar 23, 2015 at 12:50:14PM -0700, Matt Davis wrote:
> I asked @gvwilson about this and he indicated that the bottleneck
> for new instructors is not their hesitancy, but that we have more
> instructors coming on board than we have workshops in which to
> schedule them.

So when do the “Train the Admin” classes start?